docs/docs/api.md

```diff
@@ -0,0 +1,357 @@
+---
+title: REST API
+---
+
+# REST API
+
+You can communicate with Chronos using a RESTful [JSON][] API over HTTP.
+Chronos nodes usually listen on `port 8080` for API requests.
+All examples in this section assume that you've found a running leader at `chronos-node.airbnb.com:8080`.
+
```

Comment:
Also, trying to steer clear of Mesosphere mentions since this repo is technically owned by the Mesos organisation.